core/tests/behavior/write.rs

```diff
@@ -179,6 +180,49 @@ pub async fn test_write_with_special_chars(op: Operator) -> Result<()> {
     Ok(())
 }
 
+// Write a single file with cache control should succeed.
+pub async fn test_write_with_cache_control(op: Operator) -> Result<()> {
+    if !(op.info().can_presign() && op.info().capability().write_with_cache_control) {
+        return Ok(());
+    }
+
+    let path = uuid::Uuid::new_v4().to_string();
+    let (content, _) = gen_bytes();
+
+    let target_cache_control = "max-age=3600";
+
+    let mut op_write = OpWrite::default();
+    op_write = op_write.with_cache_control(target_cache_control);
+
+    op.write_with(&path, op_write, content.clone()).await?;
+
+    let signed_req = op
+        .presign_stat(&path, Duration::from_secs(60))
```

Comment:
Oh, sorry for my misleading review. I mean we can test cache-control via op.stat() directly. (Maybe we should add cache control in metadata first?)

Reply:
The `cache-control` doesn't seem like something a user would use directly. 👀